content-src/asrouter/templates/SimpleBelowSearchSnippet/SimpleBelowSearchSnippet.schema.json

```diff
@@ -29,6 +29,11 @@
       "type": "string",
       "description": "Snippet icon. Dark theme variant. 64x64px. SVG or PNG preferred."
     },
+    "icon_alt_text": {
+      "type": "string",
+      "description": "Alt text for accessibility",
```

Comment:
Maybe "Alt text describing icon for screen readers"

Reply:
that sounds perfect, I'll do that